hyperSpec/vignettes/fileio.Rmd

```diff
@@ -1259,8 +1259,9 @@ to stop with an error, but it is polite to issue an additional warning.
 writeLines(readLines("read.txt.PerkinElmer.R"))
 ```
 
+imports the spectra.
```

Comment:
Random sentence?

Reply:
![image](https://user-images.githubusercontent.com/12725868/90756600-41599700-e2e5-11ea-8db6-27a5c9ebb3f9.png)

It was not obvious in git diff.